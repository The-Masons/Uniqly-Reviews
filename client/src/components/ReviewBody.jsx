import React from 'react';
import ReactDOM from 'react-dom';
import axios from 'axios';

let ReviewBody = (props) => {
    let date = props.reviewData.date;
    date = date.split('-');
    date = date[1] + ' / ' + date[2].slice(0, 2) + ' / ' + date[0];

    return (
        <div className='review-body'>
            <div className='review-date'>{date}</div>
            <p>testing</p>
            <div className='review-title'><b>{props.reviewData.review_title}</b></div>
            <div>{props.reviewData.review_body}</div>
            <div className='review-helpful'><span className='review-helpful-text'>Was this review helpful?</span>
                <span><button className='button-thumbs'><img src="https://i.imgur.com/7RPAVLL.png"></img></button></span> <span><button className='button-thumbs'><img src='https://i.imgur.com/mip0AGG.png'></img></button></span>
            </div>
        </div>
    )
}

export default ReviewBody;