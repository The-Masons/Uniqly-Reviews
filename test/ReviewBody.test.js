import React from 'react';
import ReactDOM from 'react-dom';
import ReviewBody from '../client/src/components/ReviewBody.jsx';
import renderer from 'react-test-renderer';
import sampleReviews from '../database/data.js';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<ReviewBody reviewData={sampleReviews.sampleReviews[0]}/>, div);
});

describe('ReviewBody', () => {
  test('should render with the correct items', () => {
    const mounted = mount(
      <ReviewBody
        reviewData={sampleReviews.sampleReviews[0]}/>
      );
    expect(mounted).toMatchSnapshot();
  });
});