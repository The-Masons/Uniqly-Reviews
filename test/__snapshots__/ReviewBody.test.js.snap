// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewBody should render with the correct items 1`] = `[Function]`;
